.editorconfig

```diff
@@ -0,0 +1,15 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[{*.md,*.snap}]
+trim_trailing_whitespace = false
+
+[package.json]
+indent_size = 2
```

Comment:
Missing new line at the end of the file here and in other files as well. There's an option in your editor for this. Alternatively, install editorconfig plugin in your editor.